lib/arraystats/class.c

```diff
@@ -368,16 +352,9 @@ double AS_class_discont(double *data, int count, int nbreaks,
                 nmax = k;
             }
             nd--; /* A VERIFIER! */
```

Comment:
That's in French, means "to check", "to verify" as in a todo.

Reply:
I removed that comment, not needed anymore. Thanks!